backend/src/main/java/com/github/martingaston/application/App.java

```diff
@@ -47,25 +48,16 @@ public void listen() throws IOException {
             Request request = RequestParser.from(client);
             Routes routes = new Routes();
 
-            routes.post(URI.from("/echo_body"), (req, res) -> res.send(req.body()));
+            routes.get("/api/startGame", (req, res) -> res.json(API.startGame()));
 
-            routes.head(URI.from("/get_with_body"));
-
-            routes.get(URI.from("/simple_get"));
-
-            routes.get(URI.from("/method_options"));
-
-            routes.get(URI.from("/method_options2"));
-
-            routes.post(URI.from("/method_options2"));
-
-            routes.put(URI.from("/method_options2"));
-
-            routes.get(URI.from("/redirect"), (req, res) -> {
-                URI location = URI.from("http://" + req.getHeader("Host") + "/simple_get");
-                return res.redirect(Status.MOVED_PERMANENTLY, location);
+            routes.post("/api/getNextTurn", (req, res) -> {
+                String gameRequestAsJson = req.body().toString();
+                String ticTacToeResult = API.getNextTurn(gameRequestAsJson);
```

Comment:
This looks good 👍 
API sounds like a really weird name to me, could it be more specific?